src/test/kotlin/no/nav/familie/ef/sak/repository/SakRepositoryTest.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.familie.ef.sak.no.nav.familie.ef.sak.repository
+
+import no.nav.familie.ef.sak.OppslagSpringRunnerTest
+import no.nav.familie.ef.sak.repository.CustomRepository
+import no.nav.familie.ef.sak.repository.SakRepository
+import no.nav.familie.ef.sak.repository.domain.Barn
+import no.nav.familie.ef.sak.repository.domain.Sak
+import no.nav.familie.ef.sak.repository.domain.Søker
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.TestPropertySource
+import java.time.LocalDate
+
+@ActiveProfiles("local", "mock-auth", "mock-oauth")
+@TestPropertySource(properties = ["FAMILIE_INTEGRASJONER_URL=http://localhost:28085"])
+internal class SakRepositoryTest : OppslagSpringRunnerTest(){
+
+    @Autowired lateinit var sakRepository: SakRepository
+    @Autowired lateinit var customRepository: CustomRepository<Sak>
+
```

Comment:
Kanskje teste på ingen treff, og legge til to (eller flere) (liknende) entries før du sjekker at du får akkurat det ene treffet du er på jakt etter.

Reply:
Check